main.nf

```diff
@@ -458,7 +470,6 @@ process picard {
  */
 
 process countstat {
-    tag "${input[0].baseName}"
```

Comment:
How come you removed these?

Reply:
Hi, this process summarizes the read count for all samples. But the tag will only show the first sample which may be misleading.

Reply:
True - the reasoning for doing this elsewhere is because of running multiple instances of the pipeline on multiple projects. It can then be useful to see which project a job is from when looking in the job list.

Reply:
OK, I understand. I will put them back then.